python/tvm/autotvm/tuner/xgboost_cost_model.py

```diff
@@ -312,9 +312,15 @@ def _get_feature(self, indexes):
             for i, fea in zip(need_extract, feas):
                 fea_cache[i] = fea
 
-        ret = np.empty((len(indexes), fea_cache[indexes[0]].shape[-1]), dtype=np.float32)
+        feature_len = None
+        for idx in indexes:
+            if fea_cache[idx] is not None:
+                feature_len = fea_cache[idx].shape[-1]
```

Comment:
Is `fea_cache[idx].shape[-1]` the same for all non-None element in `fea_cache`?
Is `break` missing here?

Reply:
Good catch. A `break` is missing